### ISSUE TITLE
Running local from visual studio issue: Cannot find module '...\double_edge.js'

### ISSUE THREAD

**Issue by tjorw** [Azure/Azure-Functions#650]
Running on Windows 10, VS 15.4.5.

* I hit F5
- CLI launches and disappears
![image](https://user-images.githubusercontent.com/20905784/34772533-85edaf9a-f608-11e7-80a8-4b37657cf504.png)

- Seems like it can't find double_edge.js
- The path to double_edge.js might be confused of the swedish character "Ä" in my username. Its replaced with an "?".
* The file double_edge.js is actually installed in the right path:
 
![image](https://user-images.githubusercontent.com/20905784/34772305-df893df4-f607-11e7-9c5e-9ef66980d20a.png)

**Comment by mcraiha** [comment]
I have same issue.

```
Debugger listening on [::]:5858
module.js:471
    throw err;
    ^

Error: Cannot find module 'C:\Users\KaarloR�ih�\AppData\Roaming\npm\node_modules\azure-functions-core-tools\bin\edge\double_edge.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:389:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:504:3
```

**Comment by CYBERRRR** [comment]
Same here.

**Comment by pauer24** [comment]
I have the same problem. For the ones that are stuck with this, the template with .NET Core is working fine.

**Comment by Gismork** [comment]
I had the same issue.
Tools where installed in my user profile folder : C:\Users\Cédric\AppData\Local\Azure.Functions.Cli

I suspected the non ASCII character in the path to cause this issue.
I moved the folder elsewhere and added the new path to my PATH environment variable : D:\Temp\Azure.Functions.Cli\1.0.9

After that I was able to manually launch the tool using the "func host start" command from my Azure function debug folder.
